### ISSUE TITLE
navbar is not customizable

### ISSUE THREAD

**Issue by ppKrauss** [mkdocs/mkdocs#1211]
As commented before at #1207 is *impossible* to customize directally &mdash; without *cost of hours of work*, that is not the aim of a typical mkdocs user  &mdash;, in ideal "only change a property" tasks... So the real solution is to copy all theme (ex. [this copied theme](https://github.com/okfn-brasil/miniguias/tree/master/okbr_theme)).

The standard theme have a *div* tag and *class* attribute pairs for customize the `navbar`... But it is a mixed code immerse in the bootstrap CSS code. Ideal is to express all typical "customizable code" in the `base.css`.

PS: the issue #843  is cited in the `base.css`, but it is not  rationale to create a barrier to an easy-to-customize *navbar*.

**Comment by waylan** [comment]
Pull Requests which improve the front end (CSS, JavaScript, etc) are welcome.

**Comment by ppKrauss** [comment]
Hi @waylan, thanks the feedback and envitation.  

I know CSS but never edited an [ugly `bootstrap-custom.min.css`](https://github.com/mkdocs/mkdocs/blob/master/mkdocs/themes/mkdocs/css/bootstrap-custom.min.css)... There are some step before edit it?  Example: you generated it from [Sass](http://sass-lang.com/) or an online customizer?

If we can change it to some human-readable form, I can edit the CSS... Perhaps transform it into a template (Mustache? what you are using?)

**Comment by waylan** [comment]
The CSS was generated long before I joined the project. I'm assuming it was generated by bootstrap's [online generator][1]. Unfortunately noone seems to have the settings that were fed into the generator. Or maybe that applies to the [mkdocs-bootstrap theme][3].

Perhaps the MkDocs theme was built off of one of the [bootswatch][2] themes, which are all built on top of bootstrap.

I figured this out at one point but my memory is a little fuzzy now. In either case, the uncompressed CSS never existed as part of MkDocs but came from the third party CSS framework. Those frameworks expect that you include their compressed CSS unaltered and add your own CSS file to override/alter/add to the styles using the "cascading" part of CSS. Sounds like you are proposing making alterations to the framework, which is not likely to be accepted.

None of the current maintainers claim to be front-end experts and we seem to have just taken the defaults from a Framework and tweaked things here and there.

[1]: http://getbootstrap.com/customize/
[2]: https://bootswatch.com/
[3]: https://github.com/mkdocs/mkdocs-bootstrap
